### ISSUE TITLE
Empty Terraform Plan view

### ISSUE THREAD

**Issue by aimar-worth** [charleszipp/azure-pipelines-tasks-terraform#237]
Hello,

Sorry if I ask a silly question but is terraform plan view in Azure DevOps not ready or why I'm not able to see anything in there?

![image](https://user-images.githubusercontent.com/46780510/109778841-93209980-7c05-11eb-8095-270bf2783474.png)

My task is as following
```
- task: TerraformCLI@0
  displayName: 'terraform plan'
  inputs:
      command: plan
      environmentServiceName: xxx-xxx-xxx-$(workspace)
      commandOptions: '-out=$(System.DefaultWorkingDirectory)/terraform.tfplan -detailed-exitcode'
      workingDirectory: deployment
      allowTelemetryCollection: false
      publishPlanResults: true
```

Should this Terraform Plan view pick this up somehow or should I publish this some way?
Didn't find anything from the documentation or from the issues.

Thanks!

**Comment by charleszipp** [comment]
Hi @aimar-worth, this feature is in active development and was partially released a with 0.6.9. Unfortunately, there is no way to feature flag the views to prevent them from showing when the feature is not engaged. Therefore, you are seeing the empty view.

The feature should be finished and released this week with 0.6.10. Here's a quick preview of what to expect (in case others have same question).

When running plan via the cli, `publishPlanResults` will be set with a string that represents a name for the plan on the view. `publishPlanResults: my-plan-for-dev-env`

Then the following would appear
![image](https://user-images.githubusercontent.com/4292559/109815046-d292ba00-7bf4-11eb-942c-f1a161b6c418.png)

If no plan was published then this will appear (the message will get updated to reflect how publishPlanResults should be set)
![image](https://user-images.githubusercontent.com/4292559/109815009-c575cb00-7bf4-11eb-97f7-667ebb5e6034.png)

The summary in your screenshot will not be provided with the first release. Initially it will just be a rendering of the plan output

**Comment by charleszipp** [comment]
Also note the drop-down menu in the view. This was provided to support multiple plans being run within the same pipeline run. The dropdown will allow the user to flip between the different plans published.

**Comment by charleszipp** [comment]
cc'ing @favoretti as they are also patiently awaiting the release of this feature

**Comment by aimar-worth** [comment]
Thanks for the update @charleszipp, much appreciated!

**Comment by charleszipp** [comment]
Closing as this will be resolved once 0.6.10 is released.
